cmd/datamon/cmd/cli_test.go

```diff
@@ -30,6 +30,8 @@ const (
 	destinationDir    = "../../../testdata/cli"
 	sourceData        = destinationDir + "/data"
 	consumedData      = destinationDir + "/downloads"
+	configDirectory = destinationDir + "/config"
+	configPath = configDirectory + "/" + configFile + ".yaml"
```

Comment:
ok. Just for you to know, this I believe better captures the intent (for future refactoring actions):
```
configPath = filepath.Join(configDirectory, configFile+".yaml")
```

Reply:
this is, in part, me being periodically confused about `const` in golang as distinct from C